DAO.md

```diff
@@ -194,8 +194,7 @@ class StarWarsFilm(id: EntityID<Int>) : IntEntity(id) {
     ...
 }
 ```
-Note: Creating the entity and the reference in the same `transaction` does only work if you set the
-id column manually. If you're using `UUIDTables` and `UUIDEntity` you can do it like this:
+Note: If you're using `UUIDTables` and `UUIDEntity` you can setup IDs manually inside transaction like this:
```

Comment:
Nit: It's not just when using `UUIDTable` and `UUIDEntity`, so we can reword this.
```suggestion
Note: You can set up IDs manually inside a transaction like this:
```